src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/BuildAssetsUtils.cs

```diff
@@ -172,7 +173,10 @@ public static XDocument GenerateMultiTargetFrameworkWarning()
                             GenerateProperty("NuGetPackageRoot", ReplacePathsWithMacros(repositoryRoot)),
                             GenerateProperty("NuGetPackageFolders", string.Join(";", packageFolders)),
                             GenerateProperty("NuGetProjectStyle", projectStyle.ToString()),
-                            GenerateProperty("NuGetToolVersion", MinClientVersionUtility.GetNuGetClientVersion().ToFullString())));
+                            GenerateProperty("NuGetToolVersion", MinClientVersionUtility.GetNuGetClientVersion().ToFullString())),
+                new XElement(Namespace + "ItemGroup",
+                            new XAttribute("Condition", $" {ExcludeAllCondition} "),
```

Comment:
Why is exclude condition surrounded with spaces in Condition attribute value? They are not necessary.

Reply:
We add for all of our msbuild goo, I mostly left it there for consistency. I don't think there'd be any consequences though.

Reply:
Nope, there is no effect on evaluation. I was just curious 🙂